webhook/aws/sns_handler_test.go

```diff
@@ -52,19 +47,17 @@ const (
   } }`
 )
 
-func SetUpTestSNSServer() (*SNSServer, *mockSVC, *mux.Router)  {
+func SetUpTestSNSServer() (*SNSServer, *MockSVC, *mux.Router)  {
 	
 	v := SetUpTestViperInstance(TEST_AWS_CONFIG)
 	
-	awsCfg, err := NewAWSConfig(v.Sub(AWSKey))
-	m := &mockSVC{}
+	awsCfg, _ := NewAWSConfig(v.Sub(AWSKey))
+	m := &MockSVC{}
 	
 	ss := &SNSServer{
 		Config: *awsCfg,
 		SVC: m,
 	}
-	fmt.Println(ss)
-	fmt.Println(err)
 	
 	selfURL := &url.URL{
 		Scheme:   "http",
```

Comment:
8090 port was just example and not actually used in unit tests to start server. selfurl is used to subscribe to SNS as endpoint to receive notifications.